gui/wxpython/datacatalog/infomanager.py

```diff
@@ -0,0 +1,31 @@
+
+import wx
+import webbrowser
+
+from grass.script import gisenv
+
+
+class InfoManagerDataCatalog:
+    """
+    InfoBar Manager for Data Catalog
+    """
```

Comment:
It is a manager, not a data catalog:

```suggestion
class DataCatalogInfoManager:
    """Manager for all things info bar in Data Catalog"""
```